### ISSUE TITLE
fabio 1.3.6 UI displays host and path as 'undefined' in the routes page

### ISSUE THREAD

**Issue by bluen** [eBay/fabio#217]
Seems to affect the ui/api only - the actual routing seems to work...

The following shows the routing entries of 3 docker containers "nginxdemos/hello:latest" registered
in consul with tags `urlprefix-hello.service.consul:9999/`, `green` and `latest` for fabio-1.3.5 and fabio-1.3.6 (running in docker containers) visible in the browser.

fabio 1.3.5 output of http://fabio:9998/api/routes 
```
[
    {
        "service": "hello",
        "host": "hello.service.consul:9999",
        "path": "/",
        "dst": "http://10.7.0.23:48224/",
        "weight": 0.3333333333333333,
        "tags": [
            "green",
            "urlprefix-hello.service.consul:9999/",
            "latest"
        ],
        "cmd": "route add hello hello.service.consul:9999/ http://10.7.0.23:48224/ weight 0.33 tags \"green,urlprefix-hello.service.consul:9999/,latest\"",
        "rate1": 0,
        "pct99": 0
    },
    {
        "service": "hello",
        "host": "hello.service.consul:9999",
        "path": "/",
        "dst": "http://10.7.0.22:29747/",
        "weight": 0.3333333333333333,
        "tags": [
            "green",
            "urlprefix-hello.service.consul:9999/",
            "latest"
        ],
        "cmd": "route add hello hello.service.consul:9999/ http://10.7.0.22:29747/ weight 0.33 tags \"green,urlprefix-hello.service.consul:9999/,latest\"",
        "rate1": 0,
        "pct99": 0
    },
    {
        "service": "hello",
        "host": "hello.service.consul:9999",
        "path": "/",
        "dst": "http://10.7.0.21:23652/",
        "weight": 0.3333333333333333,
        "tags": [
            "green",
            "urlprefix-hello.service.consul:9999/",
            "latest"
        ],
        "cmd": "route add hello hello.service.consul:9999/ http://10.7.0.21:23652/ weight 0.33 tags \"green,urlprefix-hello.service.consul:9999/,latest\"",
        "rate1": 0,
        "pct99": 0
    }
]
```

fabio 1.3.6 of the exact same:
```
[
    {
        "cmd": "route add",
        "service": "hello",
        "src": "hello.service.consul:9999/",
        "dst": "http://10.7.0.23:48224/",
        "weight": 0.3333333333333333,
        "tags": [
            "green",
            "urlprefix-hello.service.consul:9999/",
            "latest"
        ],
        "rate1": 0,
        "pct99": 0
    },
    {
        "cmd": "route add",
        "service": "hello",
        "src": "hello.service.consul:9999/",
        "dst": "http://10.7.0.22:29747/",
        "weight": 0.3333333333333333,
        "tags": [
            "green",
            "urlprefix-hello.service.consul:9999/",
            "latest"
        ],
        "rate1": 0,
        "pct99": 0
    },
    {
        "cmd": "route add",
        "service": "hello",
        "src": "hello.service.consul:9999/",
        "dst": "http://10.7.0.21:23652/",
        "weight": 0.3333333333333333,
        "tags": [
            "green",
            "urlprefix-hello.service.consul:9999/",
            "latest"
        ],
        "rate1": 0,
        "pct99": 0
    }
]
```

I think the ui (javascript) tries to read the values for "host" and "path" from the JSON fields with the same name - but in 1.3.6 those fields do not exist (but the values can be parsed from the "src" field")

**Comment by bluen** [comment]
Screen shots:
fabio-1.3.5 - ![fabio-1 3 5](https://cloud.githubusercontent.com/assets/7766604/22061998/c32bd80a-dd77-11e6-8eca-a91127e56d12.png)
fabio-1.3.6 - ![fabio-1 3 6](https://cloud.githubusercontent.com/assets/7766604/22062000/c4b1c5fe-dd77-11e6-8300-0664e16ff883.png)

**Comment by magiconair** [comment]
Yes, I made a boo-boo. master has the fix already. Sorry about that.

**Comment by magiconair** [comment]
Released 1.3.7
